### ISSUE TITLE
mautrix-telegram shows several error messages in asyncpg while building from source

### ISSUE THREAD

**Issue by 81909ce2298f99ddafa37936ba43eaac** [mautrix/telegram#986]
I tried to install mautrix-telegram in a virtual environment by using `pip install --upgrade mautrix-telegram[all]`. While this worked before updating the system (Void Linux), the following errors now occurs when compiling "asyncpg":

```
...
asyncpg/pgproto/pgproto.c: In function '__Pyx_PyInt_As_int32_t':
asyncpg/pgproto/pgproto.c:42239:27: error: too few arguments to function '_PyLong_AsByteArray'
42239 |                 int ret = _PyLong_AsByteArray((PyLongObject *)v,
           |                           ^~~~~~~~~~~~~~~~~~~
/usr/include/python3.13/cpython/longobject.h:111:17: note: declared here
111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
       |                 ^~~~~~~~~~~~~~~~~~~
asyncpg/pgproto/pgproto.c: In function '__Pyx_PyInt_As_long':
asyncpg/pgproto/pgproto.c:42435:27: error: too few arguments to function '_PyLong_AsByteArray'
42435 |                 int ret = _PyLong_AsByteArray((PyLongObject *)v,
           |                           ^~~~~~~~~~~~~~~~~~~
/usr/include/python3.13/cpython/longobject.h:111:17: note: declared here
111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
       |                 ^~~~~~~~~~~~~~~~~~~
asyncpg/pgproto/pgproto.c: In function '__Pyx_PyInt_As_unsigned_char':
asyncpg/pgproto/pgproto.c:42745:27: error: too few arguments to function '_PyLong_AsByteArray'
42745 |                 int ret = _PyLong_AsByteArray((PyLongObject *)v,
           |                           ^~~~~~~~~~~~~~~~~~~
/usr/include/python3.13/cpython/longobject.h:111:17: note: declared here
        111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
            |                 ^~~~~~~~~~~~~~~~~~~
asyncpg/pgproto/pgproto.c: In function '__Pyx_PyInt_As_int16_t':
asyncpg/pgproto/pgproto.c:43055:27: error: too few arguments to function '_PyLong_AsByteArray'
43055 |                 int ret = _PyLong_AsByteArray((PyLongObject *)v,
           |                           ^~~~~~~~~~~~~~~~~~~
/usr/include/python3.13/cpython/longobject.h:111:17: note: declared here
111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
       |                 ^~~~~~~~~~~~~~~~~~~
asyncpg/pgproto/pgproto.c: In function '__Pyx_PyInt_As_char':
asyncpg/pgproto/pgproto.c:43289:27: error: too few arguments to function '_PyLong_AsByteArray'
43289 |                 int ret = _PyLong_AsByteArray((PyLongObject *)v,
           |                           ^~~~~~~~~~~~~~~~~~~
/usr/include/python3.13/cpython/longobject.h:111:17: note: declared here
111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
       |                 ^~~~~~~~~~~~~~~~~~~
asyncpg/pgproto/pgproto.c: In function '__Pyx_PyInt_As_int8_t':
asyncpg/pgproto/pgproto.c:43485:27: error: too few arguments to function '_PyLong_AsByteArray'
43485 |                 int ret = _PyLong_AsByteArray((PyLongObject *)v,
           |                           ^~~~~~~~~~~~~~~~~~~
/usr/include/python3.13/cpython/longobject.h:111:17: note: declared here
111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
       |                 ^~~~~~~~~~~~~~~~~~~
asyncpg/pgproto/pgproto.c: In function '__Pyx_PyInt_As_uint32_t':
asyncpg/pgproto/pgproto.c:43719:27: error: too few arguments to function '_PyLong_AsByteArray'
43719 |                 int ret = _PyLong_AsByteArray((PyLongObject *)v,
           |                           ^~~~~~~~~~~~~~~~~~~
/usr/include/python3.13/cpython/longobject.h:111:17: note: declared here
111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
       |                 ^~~~~~~~~~~~~~~~~~~
asyncpg/pgproto/pgproto.c: In function '__Pyx_PyInt_As_uint64_t':
asyncpg/pgproto/pgproto.c:43915:27: error: too few arguments to function '_PyLong_AsByteArray'
43915 |                 int ret = _PyLong_AsByteArray((PyLongObject *)v,
           |                           ^~~~~~~~~~~~~~~~~~~
/usr/include/python3.13/cpython/longobject.h:111:17: note: declared here
111 | PyAPI_FUNC(int) _PyLong_AsByteArray(PyLongObject* v,
       |                 ^~~~~~~~~~~~~~~~~~~
...
```

Python 3.13.1 is used on the system.

Note: 
Mautrix-telegram shows the message "Collecting asyncpg<0.30,>=0.20 (from mautrix-telegram[all])" while building from source, but asyncpg 0.30 is the latest version.

**Comment by tulir** [comment]
You can override it using `pip install asyncpg>=0.30`

**Comment by LelieL91** [comment]
> You can override it using `pip install 'asyncpg>=0.30'`

Unfortunatelly this approach doesn't work, I already tried it, even if installed before telegram bridge, it still tries to build wheels using 0.29 as requirements.txt leading in a fail.

Today I just discoverd I wasn't the only one getting this issue, but since it was closed, like mine, I guess it's kind tricky to find out.

@81909ce2298f99ddafa37936ba43eaac I kind fixed it on Arch OS that it is using your same Python version, by installing the telegram bridge from the master branch instead of latest stable as described in a line on the [Docs](https://docs.mau.fi/bridges/python/setup.html) at 2nd step of Production setup.

This is the line I used: `pip install --no-cache-dir --upgrade mautrix-telegram[all]@git+https://github.com/mautrix/telegram.git`
